.github/workflows/build_and_test.yml

```diff
@@ -245,10 +253,17 @@ jobs:
 
   build-image-pr:
     runs-on: ubuntu-latest
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && env.GITHUB_HEAD_REF == ''
 
     steps:
-    - uses: actions/checkout@v1
+    - name: DEBUG
+      run: |
+        echo "env"
+        echo $GITHUB_HEAD_REF
+        echo "expression"
+        echo ${{ github.head_ref }}
+
```

Comment:
Is this intended to stay?

Reply:
no.. the PR is not working yet, when it works, I'll remove the debug step ;)